recipes/blend_modes/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "blend_modes" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a3145e4792e005764b9663f5ce899d30f7c24f4bcff00428907d03dbe068f37
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
can this be noarch python?

Reply:
Yes. The only Python version requirement is >= 3.5.

I followed the [maintainer knowledge base on conda-forge.org](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds) which mentions explicitly as noarch requirement:
> No skips except for python version. If the recipe is py3 only, remove skip statement and add version constraint on python in host and run section.

I removed the skip statement and added version constraints here:

https://github.com/conda-forge/staged-recipes/blob/e9c968f236e38e620252a0ccec7cc19f88a6981c/recipes/blend_modes/meta.yaml#L17-L25